features/generate.feature

```diff
@@ -64,11 +64,26 @@ Scenario: The project is an iOS application with a target dependency and transit
     Then the product 'App.app' with destination 'Debug-iphoneos' contains the framework 'Framework2' without architecture 'x86'
     Then I should be able to build the scheme Framework1
 
-Scenario: The project is an iOS application that has resources (ios_app_with_framework_and_resources)
+  Scenario: The project is an iOS application that has resources (ios_app_with_framework_and_resources)
     Given that tuist is available
     And I have a working directory
     Then I copy the fixture ios_app_with_framework_and_resources into the working directory
     Then tuist generates the project
     Then I should be able to build the scheme App
     Then the product 'App.app' with destination 'Debug-iphoneos' contains resource 'tuist.png'
     Then the product 'App.app' with destination 'Debug-iphoneos' does not contain resource 'do_not_include.dat'
+
+  Scenario: The project is an iOS application that has an app and multiple frameworks setup inside of a custom workspace (ios_app_with_custom_workspace)
```

Comment:
What about merging this into other existing scenario? I think these tests are of high value but we shouldn't disregard that they are not unit test and thus take longer to execute. If we can aggregate some of them together I think we should.

Reply:
Acceptance tests also have an extra maintenance cost so the more fixture projects we have, the more we'll have to update when we introduce breaking changes in Tuist.